lib/validators/changeset.js

```diff
@@ -0,0 +1,20 @@
+const { Validator } = require('./validator')
+
+class Changeset extends Validator {
+  constructor () {
+    super('changeset')
+    this.supportedEvents = [
+      'pull_request.*'
```

Comment:
Do we need `pull_request_reviews.*` here since you listed that in the README?

Reply:
that's actually a great catch, fixed

Reply:
Did you push your changes? I don't see it.

Reply:
sorry, i did attempt a push but it was rejected, i should've double checked that the push went thru. I fixed it